### ISSUE TITLE
Using other vendors' emojis (like Google's)?

### ISSUE THREAD

**Issue by Tymewalk** [Tymewalk/Tymoji#17]
In [Unicode's full emoji list](http://unicode.org/emoji/charts/full-emoji-list.html) major vendors like Google, Facebook Messenger, and EmojiOne all are listed.

I'd like to use some like Google's, but I'm concerned about whether or not it would be legal to do so. I already use EmojiOne, but they're licensed under Creative Commons, which means I can do so.

**Comment by herohamp** [comment]
Twitter emojis are open source

**Comment by herohamp** [comment]
http://twitter.github.io/twemoji/preview.html

**Comment by herohamp** [comment]
http://twitter.github.io/twemoji/

**Comment by Tymewalk** [comment]
@herohamp Nice! I'm working on adding that now. It looks like the filepaths should be just about the same as those of EmojiOne.

**Comment by Tymewalk** [comment]
56b9e29 adds Twitter as an emoji set. Twitter's emoji sets have some problems with missing PNGs but I've created an issue to resolve that.

**Comment by Tymewalk** [comment]
Google has their emojis on GitHub [here](https://github.com/googlei18n/noto-emoji). I'm still a little unsure as to whether or not we'd legally be allowed to - can @herohamp or @joker314 confirm?

**Comment by Tymewalk** [comment]
[This](http://stackoverflow.com/a/16696512/) says that Google licenses their Android emoji font under Apache 2, so we should be able to use it.

**Comment by Tymewalk** [comment]
Bumping. Could someone confirm we'd be allowed to use Google's emojis?

**Comment by joker314** [comment]
We might also want to let the user select their own emojis. They can upload them and have them as a data-url.

**Comment by joker314** [comment]
Please can you give me the status of "collaborator"? Unless you don't want to..

**Comment by joker314** [comment]
I don't think we can use their emoji, but I can write them a quick email if I have time. Would that be alright with you guys?

**Comment by Tymewalk** [comment]
@joker314 Sure, I'll add you in.

Also writing an e-mail is fine. :dagger:
